.github/workflows/ci-cd.yml

```diff
@@ -40,11 +40,23 @@ jobs:
           go-version: 1.17.x
       - name: Check out source code
         uses: actions/checkout@v1
-      - name: Install dependencies
+      - name: Cache go dependencies
```

Comment:
did this improve completion time?

Reply:
There is an improvement, albeit a small one. With this change, the dependencies install in around 1 minute 45 seconds, and without it, it takes around 2:38 to 2:53 minutes.